tools/packaging/kata-deploy/README.md

```diff
@@ -43,6 +43,34 @@ $ kubectl apply -f kata-rbac/base/kata-rbac.yaml
 $ kubectl apply -k kata-deploy/overlays/rke2
 ```
 
+#### [k0s] cluster
+
+For your [k0s](https://k0sproject.io/) cluster, run:
+
+```sh
+$ git clone https://github.com/kata-containers/kata-containers.git
+```
+
+Check and switch to the stable branch of your choice, if wanted, and then run:
+
+```bash
+$ cd kata-containers/tools/packaging/kata-deploy
+$ kubectl apply -f kata-rbac/base/kata-rbac.yaml
+$ kubectl apply -k kata-deploy/overlays/k0s
+```
+
+##### Note
+
+The supported version of k0s is **v1.27.1+k0s** and above, since the k0s support leverages a special dynamic containerd configuration mode:
+
+> From 1.27.1 onwards k0s enables dynamic configuration on containerd CRI runtimes. This works by k0s creating a special directory in /etc/k0s/containerd.d/ where user can drop-in partial containerd configuration snippets.
+> 
+> k0s will automatically pick up these files and adds these in containerd configuration imports list. If k0s sees the configuration drop-ins are CRI related configurations k0s will automatically collect all these into a single file and adds that as a single import file. This is to overcome some hard limitation on containerd 1.X versions. Read more at containerd#8056
+
+However, this would also require a magic string set in the beginning of the line. Whatever the cases, the autogenerated Kata containerd configuration would always be there at `/etc/k0s/containerd.d/kata-containers.toml`.
+
+If you cannot meet the version requirement or disabled the automatic config management, you could also choose to manually append the config `kata-containers.toml` into your own config.toml
```

Comment:
Not really, the daemonset will actually replace any manual configuration the user does.  So, I'd drop this part.

Reply:
Yeah. I'd still drop this paragraph and the one above.  If the conditions are not met, they are not met, and you don't rely on  kata-deploy.  That's better than telling folks to do manual steps, which may end up overriden by the daemonset.